models/src/main/java/com/vimeo/networking2/LiveQuota.kt

```diff
@@ -13,18 +13,25 @@ import com.vimeo.networking2.annotations.Internal
 @JsonClass(generateAdapter = true)
 data class LiveQuota(
 
-    /**
-     * Live streams quota data.
-     */
-    @Internal
-    @Json(name = "streams")
-    val streams: LiveStreamsQuota? = null,
+        /**
+         * Live streams quota data.
+         */
+        @Internal
+        @Json(name = "streams")
+        val streams: LiveStreamsQuota? = null,
 
-    /**
-     * Live time data.
-     */
-    @Internal
-    @Json(name = "time")
-    val time: LiveTime? = null
+        /**
+         * Live time data.
+         */
+        @Internal
+        @Json(name = "time")
+        val time: LiveTime? = null,
+
+        /**
+         * Live status data.
+         */
+        @Internal
+        @Json(name = "status")
+        val status: String? = null
```

Comment:
Design documents indicate that this is an enum with values
- `available`
- `private_mode`
- `stream_limit`
- `time_limit`

Based on other enums in the library, we should create an enum named `StatusType` or `LiveQuotaStatusType` with the above values. Then an extension function should be created in this file as

```kotlin
val LiveQuota.statusType: StatusType
  get() = status.asEnum(StatusType.UNKNOWN)
```

See `Live` or `Privacy` for examples of how this enum is implemented.

Additionally, the legacy vimeo-networking should be updated to include the enum as well.